AndroidTest/app/src/main/java/cloudant/com/androidtest/keyprovider/ProviderTestUtil.java

```diff
@@ -0,0 +1,28 @@
+package cloudant.com.androidtest.keyprovider;
+
+import com.cloudant.sync.datastore.encryption.KeyData;
+
+import java.security.SecureRandom;
+import java.util.Random;
+import java.util.UUID;
+
+public class ProviderTestUtil {
+    public static String getUniqueIdentifier() {
+        return "test-id-" + UUID.randomUUID();
+    }
+
+    public static KeyData createKeyData() {
+        byte[] encryptedDPK = new byte[32];
+        byte[] salt = new byte[32];
+        byte[] iv = new byte[32];
+        int iterations = new Random().nextInt(100000);
+        String version = "1." + System.currentTimeMillis();
+
+        SecureRandom secureRandom = new SecureRandom();
+        secureRandom.nextBytes(encryptedDPK);
```

Comment:
Should we create the `encryptedDPK` by ciphering a pre-fixed `DPK` with `iv` and a known password? We do not need this at this moment, the current test does not require it but I was wondering if, at some point, we will need a plain DPK based on the data generated with this code.